tutorials/spa-create-automation/spa-create-automation.md

```diff
@@ -43,61 +43,61 @@ An Automation is a succession of steps to orchestrate multiple activities and ap
 
 1. In the **Lobby** from the **editable version** of your project, do the following:
     - Select the process **Order Processing**.
-    - Choose ![00-01](AddButtonProcess.png)
-    - Select **Automation** > **New Automation**.
+    - Choose **+**
+    - Select **Automation**, **New Automation**.
 
-    !![00-01](Step1.2-CreateNewAutomation.png)
+    !![001](001.png)
 
 2. A pop up will appear to configure the Desktop Agent version. Do the following in the pop up:
 
-    - From the dropdown, select the version of the Desktop Agent installed on your machine.
+    - From the dropdown, select the version of the Desktop Agent installed on your machine. It would be displayed with suffix as **Registered**.
+    - Choose **Confirm** button.
 
-    > It would be with suffix as **Registered**.
-
-    - Choose **Confirm**.
-
-    ![00-01](Step1.3-ConfigureAgentVersion.png)
+    ![001](002.png)
 
 3. A new pop-up will appear to create the automation. Do the following in the pop-up:
 
-    -  Enter **Name** of the automation as **Get Order Details**.
-    -  Enter a **Description**.
-    -  Choose **Create**.
+    -  Under Name Field enter: **Get Order Details**,
+    -  Under Description enter: **Automation for Order Process**,
+    -  Choose **Create** button.
 
     > Identifier will be auto-filled.
 
-    ![00-01](Step2-AutomationName.png)
+    ![001](003.png)
 
     An automation **Get Order Details** will be created successfully.
 
-    !![00-01](Step1.4-AutomationCreated.png)
+    !![001](004.png)
 
 [DONE]
 [ACCORDION-END]
 
 [ACCORDION-BEGIN [Step 2: ](Create Environment Variable)]
 
-Business project usually need to use parameters and variable at runtime. These variables are usually saved in their runtime landscapes, Dev, Test or Production environments for example. In this case, you will need to maintain an environment variable that will contain the file full path of the excel file used in the automation.
+Business projects usually need to use parameters and variable at runtime. These variables are usually saved in their runtime landscapes e.g., Dev, Test or Production environments. In this case, you will need to maintain an environment variable that will contain the file full path of the excel file used in the automation.
```

Comment:
"for example" was good. It was just in the wrong position.

Reply:
are we not allowed to use e.g.?

Reply:
It is not forbidden, but several programs that read webpages for those with visual impairment read ‘eg’ and other abbreviations incorrectly. For those who did not grow up with English as their first language, it may also be difficult to understand Latin abbreviations within an English text.

Reply:
ok I made changes